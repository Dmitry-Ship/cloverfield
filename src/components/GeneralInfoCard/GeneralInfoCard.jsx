import React, { PropTypes } from 'react';

import { card, field } from './GeneralInfoCard.scss';

const GeneralInfoCard = ({ tags, notes, images, className }) => (
  <div className={`${card} ${className}`} >
    <h2 className={field} >📝Notes: {notes}</h2>
    <h2 className={field} >🖼️Images: {images}</h2>
    <h2 className={field} >🏷️Tags: {tags}</h2>
  </div>
);

export default GeneralInfoCard;

GeneralInfoCard.defaultProps = {
  className: '',
};

GeneralInfoCard.propTypes = {
  tags: PropTypes.number.isRequired,
  notes: PropTypes.number.isRequired,
  images: PropTypes.number.isRequired,
  className: PropTypes.string,
};
