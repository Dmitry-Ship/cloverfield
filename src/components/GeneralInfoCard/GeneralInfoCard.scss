.card {
  composes: inofCard from 'styles/cards.scss';
  display: flex;
  justify-content: space-between;
  text-align: center;
  margin-top: 50px;
  margin-bottom: 50px;
}

.field {
  display: block;
}
