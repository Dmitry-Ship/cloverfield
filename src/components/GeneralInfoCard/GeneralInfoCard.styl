.card
  composes: inofCard from 'styles/cards.styl';
  display: flex
  justify-content: space-between;
  text-align: center;
  margin-top: 50px
  margin-bottom: 50px

.field
  display: block;
