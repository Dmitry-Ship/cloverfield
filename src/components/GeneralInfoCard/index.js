export { default } from './GeneralInfoCard.jsx';
