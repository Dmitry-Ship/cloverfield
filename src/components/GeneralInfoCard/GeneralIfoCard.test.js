import React from 'react';
import renderer from 'react-test-renderer';
import GeneralInfoCard from './GeneralInfoCard';

describe('<GeneralInfoCard />', () => {
  it('should render correctly', () => {
    const tree = renderer.create(
      <GeneralInfoCard images={1} tags={1} notes={1} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
