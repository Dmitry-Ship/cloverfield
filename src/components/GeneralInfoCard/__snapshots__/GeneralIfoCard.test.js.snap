// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GeneralInfoCard /> should render correctly 1`] = `
<div
  className="card "
>
  <h2
    className="field"
  >
    📝Notes: 
    1
  </h2>
  <h2
    className="field"
  >
    🖼️Images: 
    1
  </h2>
  <h2
    className="field"
  >
    🏷️Tags: 
    1
  </h2>
</div>
`;
